smartseq2_single_sample/tasks/hisat2.wdl

```diff
@@ -35,16 +78,18 @@ task HISAT2SE {
   File fq
   String ref_name
   String output_name
-  
+  String samplename
   command {
     tar -zxvf "${hisat2_ref}"
     hisat2 -t \
-      -x ${ref_name}/genome_snp_tran \
+      -x ${ref_name}/${ref_name} \
       -U ${fq} \
+      --rg-id=${samplename} --rg SM:${samplename} --rg LB:${samplename} \
```

Comment:
I liked how these were multi-lined in the other tasks, could you also do that here?